substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/JfrSymbolRepository.java

```diff
@@ -82,16 +82,16 @@ private JfrSymbolHashtable getTable(boolean previousEpoch) {
 
     @Uninterruptible(reason = "Epoch must not change while in this method.")
     public long getSymbolId(Class<?> clazz) {
```

Comment:
I think it would be safer to add the `boolean previousEpoch` argument to all `getSymbolId(...)` methods. That should make it clearer which epoch is used in which place.